src/tools/illink/test/Mono.Linker.Tests.Cases/DataFlow/MemberTypesAllOnCopyAssembly.cs

```diff
@@ -65,8 +65,7 @@ namespace Mono.Linker.Tests.Cases.DataFlow
 	[KeptTypeInAssembly ("base.dll", "Mono.Linker.Tests.Cases.DataFlow.Dependencies.MemberTypesAllBaseType/PrivateNestedType")]
 	[KeptMemberInAssembly ("base.dll", "Mono.Linker.Tests.Cases.DataFlow.Dependencies.MemberTypesAllBaseType/PrivateNestedType", new string[] { "PrivateMethod()" })]
 
-	// https://github.com/dotnet/runtime/issues/78752
-	[KeptMember (".ctor()", By = Tool.Trimmer)]
+	[KeptMember (".ctor()", By = Tool.Trimmer | Tool.NativeAot)]
```

Comment:
```suggestion
	[KeptMember (".ctor()")]
```

We don't have Kept validation for the analyzer (it doesn't make sense).